sql/06-compare_buildings.sql

```diff
@@ -246,6 +294,8 @@ $$
 
 BEGIN
 
+CREATE EXTENSION IF NOT EXISTS intarray;
```

Comment:
Move this line to `nz-buildings-load` with the `psql -c "CREATE EXTENSION IF NOT EXISTS postgis SCHEMA public;"` so we won't run this line repeatedly when clicking compare

Reply:
and please add the extra relationship mentioned in the issue #148 :) Thanks